src/Avalonia.Controls/Presenters/TextPresenter.cs

```diff
@@ -129,14 +147,11 @@ public override void Render(DrawingContext context)
 
                 var rects = FormattedText.HitTestTextRange(start, length);
 
-                if (_highlightBrush == null)
-                {
-                    _highlightBrush = (IBrush)this.FindResource("HighlightBrush");
-                }
+                _selectionBrush = SelectionBrush;
```

Comment:
I think this can be a local variable.

Reply:
Could you explain more?

Reply:
just change it to `var selectionBrush = SelectionBrush` and remove `_selectionBrush`

Reply:
Yes , or maybe remove _selectionBrush and just use SelectionBrush styled property?